src/drivers/nuopc/cime_driver/esm.F90

```diff
@@ -1639,4 +1551,22 @@ subroutine ReadAttributes(gcomp, config, label, relaxedflag, formatprint, rc)
 
   end subroutine ReadAttributes
 
+  subroutine InitAdvertize(driver, importState, exportState, clock, rc)
```

Comment:
Rocky - we should add a comment here about why we need this - why do we need it?

Reply:
This empty InitAdvertise is needed because it overrides the behavior of the default InitAdvertise inside the generic NUOPC_Driver.F90.  The default  behavior tries to mirror the fields up the hierarchy (i.e., up to the ensemble driver). This would be used if we needed to communicate between the ensemble members.  Since we do not need that right now, we turn it off with this empty subroutine.